src/ci/docker/x86_64-gnu-tools/checkregression.py

```diff
@@ -32,7 +35,8 @@
                 'The state of "{}" has {} from "{}" to "{}"'
                 .format(tool, verb, state, new_state)
             )
-            regressed = True
+            if tool not in REGRESSION_OK:
```

Comment:
```suggestion
            if verb == 'regressed' and tool not in REGRESSION_OK:
```

Reply:
This is a change in behavior though for the other tools, where previously `sys.exit(1)` was called if `verb == 'changed'`.

Reply:
I tried to understand when it gets called with which verb and what that means and why, but failed.^^

Reply:
@kennytm I also don't understand this change. Previously, regressed was set to true unconditionally.

Reply:
@kennytm wait a second... is it correct that the logic here requires the toolstate to *change* when a PR touches the submodule? Like, changing it from test-fail to build-fail (or test-pass to test-fail) would actually be accepted? That seems odd.^^

Reply:
Sorry, fixed the suggestion. The intended behavior is that nightly tools regressions won't cause exit(1).